### PR TITLE
Refactor `assertEquals(<boolean-literal>, …)` and `assertNotEquals(<boolean-literal>, …)` to `assertTrue` or `assertFalse`

### PR DESCRIPTION
**Pull request by ammachado**
## What's changed?

Improves the current cleanup rules by refactoring `assertEquals(false, a.methodCall(...))` to `assertFalse(a.methodCall(...))`

### Checklist
- [x] I've added unit tests to cover both positive and negative cases
- [x] I've added the license header to any new files through `./gradlew licenseFormat`
- [x] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    if (mi.getSelect() == null) {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .staticImports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
+                                .javaParser(javaParser(ctx))
+                                .build();
+                    } else {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .imports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
```
Comment (timtebeek) [inline src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java:90]: I'm wondering if this should be 
```suggestion
                                .imports("org.junit.jupiter.api.Assertions")
```

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                    if (mi.getSelect() == null) {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .staticImports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
+                                .javaParser(javaParser(ctx))
+                                .build();
+                    } else {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .imports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
```
Comment (timtebeek) [inline src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java:90]: Same as above
```suggestion
                                .imports("org.junit.jupiter.api.Assertions")
```

File: src/test/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBooleanTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 the original author or authors.
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBooleanTest.java:2]:
```suggestion
 * Copyright 2023 the original author or authors.
```

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 the original author or authors.
```
Comment (timtebeek) [inline src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java:2]:
```suggestion
 * Copyright 2023 the original author or authors.
```

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java [hunk 5]
```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 the original author or authors.
```
Comment (timtebeek) [inline src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java:2]:
```suggestion
 * Copyright 2023 the original author or authors.
```

File: src/test/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBooleanTest.java [hunk 6]
```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 the original author or authors.
```
Comment (timtebeek) [inline src/test/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBooleanTest.java:2]:
```suggestion
 * Copyright 2023 the original author or authors.
```

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java (hunk 2 above)
Comment (ammachado) [reply src/main/java/org/openrewrite/java/testing/cleanup/AssertNotEqualsBooleanToAssertBoolean.java:90]: See previous commment

File: src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java (hunk 1 above)
Comment (ammachado) [reply src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java:90]: I used `AssertEqualsNullToAssertNull` as a reference for this recipe. It adds the imports this way to keep the existing style on the refactored file (e.g.,. if the refactored file is not using static imports, the class keep it that way.

**Comment by timtebeek** [comment]
Thanks a lot @ammachado ! You can try out this change through [our snapshot versions](https://docs.openrewrite.org/reference/snapshot-instructions). We'll follow up with a release soon. I'll also push this up the the Moderne platform such that you can try it out at scale there.

**Comment by timtebeek** [comment]
Ran this again netflix conductor, with some results there: https://app.moderne.io/results/QtIfI8Wpq/details/eyJfX3R5cGVuYW1lIjoiR2l0SHViUmVwb3NpdG9yeSIsIm9yaWdpbiI6ImdpdGh1Yi5jb20iLCJwYXRoIjoiTmV0ZmxpeC9jb25kdWN0b3IiLCJicmFuY2giOiJtYWluIn0=?referrer=https%3A%2F%2Fapp.moderne.io%2Fresults%2FQtIfI8Wpq
image

Confirmed to work; thanks again! And feel free to open that PR to conductor if you're up for it. :)

**Pull request merged**